### DISCUSSION TITLE
Filters Feature : being able to hide comments in the `Files`section for better readability

### DISCUSSION THREAD

**Discussion by frantisekrokusekpa** [community/community#8549] [Pull Requests]
### Context

When there is ongoing conversations and changes requests on the Files.

### Problem statement

It is really hard to read the code with all the comments/discussion in the middle of it. Even when they are resolved!

### Solution

Add the ability to hide comments in the `Files` section

**Comment by talmewho-work** [comment]
I resort to changing the CSS to hide the comments temporarily to get some readability improvement, this is really needed.

**Comment by HugoBelotDeloro** [comment]
Please, this makes the code absolutely unreadable for me. I cannot focus on it with a big "X marked this conversation as resolved." section in the middle of a function,  otherwise my brain just parses it as the split between two files.

There is already a settings panel to select diff view style (unified or side-by-side), just add an option there, simple. Or at least don't show the conversations that have been marked as resolved, this is stupid.
